packages/google_maps_flutter/google_maps_flutter_web/pubspec.yaml

```diff
@@ -15,7 +15,8 @@ flutter:
       web:
         pluginClass: GoogleMapsPlugin
         fileName: google_maps_flutter_web.dart
-
+  assets:
+    - icons/
```

Comment:
About `blue-dot.png`: Please, remove the blue "halo" around the button, because that should use `coords.accuracy` to compute its radius. It's currently hardcoded. (We could have several versions of the marker image for different radius values, but if draw the halo with CSS, we can have animations and nicer things).